src/main/java/ua/javarush/island/entities/animals/herbivores/Boar.java

```diff
@@ -0,0 +1,26 @@
+package ua.javarush.island.entities.animals.herbivores;
+
+import ua.javarush.island.domain.Herbivore;
+import ua.javarush.island.entities.Entity;
+import ua.javarush.island.entities.plants.Plant;
+import ua.javarush.island.utilities.Config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Config(fileName = "animals/herbivores/boar.yaml")
+public class Boar extends Herbivore {
+
+    {
+        Map<Entity, Integer> eatableTable = new HashMap<>() {{
```

Comment:
do we have the same values for all instances of the class? If so, should we use static context?